apps/pie-docs/src/assets/styles/components/_card.scss

```
@@ -22,15 +26,43 @@ h2 + .c-card-wrapper {
     padding: 0;
 }
 
+.c-card-list--fillContainer {
+    grid-template-columns: 1fr;
+
+
+    @include f.media('>=mid') {
+        grid-template-columns: auto auto auto;
+
+        @container (#{$card-container-max-width}) {
```

Nice container query 😎 
What do you reckon to having it like this? Haven't tested it but to me it makes sense to have the calculation separate where we might have better linting, etc
```suggestion
        @container ((max-width: #{$card-container-max-width})) {
```

---

I thought I'd tried every possible variation of brackets and interpolation to get it working that way.
Thank you very much